pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+# Should be mirrored in requirements-build.txt
+requires = [
```

Comment:
I'm getting a bit nervous about having copies of our dependencies in workflows, requirements.txt, setup.py, and now also pyproject.toml (not to mention internal copies in internal fms). Is there some better way to keep them in sync? Are any redundant?